packages/toml/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "prettier-plugin-toml",
+  "version": "0.0.0",
```

```suggestion
  "version": "1.0.0",
```

It should sync with npm latest.